EIPS/eip-5570.md

```diff
@@ -0,0 +1,246 @@
+---
+eip: 5570
+title: Digital Receipt Non-Fungible Tokens
+description: Non-Fungible Tokens as a method of distributing digital receipts for physical purchases, where the metadata represents a json receipt
```

Comment:
Just cleaning up the wording a bit:

```suggestion
description: Non-Fungible Tokens as digital receipts for physical purchases, where the metadata represents a JSON receipt
```

---

I'm not too sure what "the metadata represents a JSON receipt" means. Do you mean something like "with JSON encoded metadata"?

Reply:
This is simply poor English on my part. Intention was to convey that an NFT has metadata, and that metadata can be a json receipt